Sources/TuistSupport/Constants.swift

```diff
@@ -27,6 +27,13 @@ public struct Constants {
         public static let swiftDoc = "swift-doc"
     }
 
+    public struct DependenciesDirectory {
+        public static let name = "Dependencies"
+        public static let graphName = "graph.json"
+        public static let lockfilesDirectoryName = "Lockfiles"
+        public static let cartfileResolvedName = "Cartfile.resolved"
+    }
```

Comment:
Can we convert it to an enum? Since everything is static, we should avoid people going `DependenciesDirectory()`

Reply:
I see that everything is a struct though 😅

Reply:
I have followed convention in this file. 

`Constants.Vendor`, `Constants.DerivedDirectory` etc

Im agree with you but I don't want to introduce a mess here.

Reply:
Basically we should convert every `struct` in this file to `enum`. Should I do it in this PR? 🤔